test/conf/protractor.local.ios.conf.js

```diff
@@ -52,16 +51,17 @@ config.multiCapabilities = [
 
 config.plugins = [
 	{
-		path: join(process.cwd(), './build/index.js'),
+		path: resolve(process.cwd(), './build/index.js'),
 		options: {
-			baselineFolder: join(process.cwd(), './localBaseline'),
+			baselineFolder: resolve(process.cwd(), './test/images/baseline'),
```

Comment:
This is needs to be the `localBaseline` that doesn't need to be submitted to the repo because it is a local test config